test-data/configuration_tests/input/jobs/snake_oil_npv.py

```diff
@@ -89,8 +89,8 @@
         production_value = oil_price * production_sum
         npv += production_value
 
-    with open("snake_oil_npv.txt", "w") as output_file:
-        output_file.write("NPV %s\n" % npv)
+    with open("snake_oil_npv.txt", "w", encoding="utf-8") as output_file:
+        output_file.write(f"NPV {npv}\n")
```

Comment:
The same as above, but here in addition it was formatted to string in the end (?!)

Reply:
Could have said `f{str(npv)}`, that would make it guaranteed to be equal to before, as I suppose we now call `__repr__` instead of `str`. I chose the simple way as I think our tests should not rely on those kind of details unless it is important in that particular context.